rotkehlchen/db/schema.py

```diff
@@ -601,7 +601,6 @@
 
 DB_CREATE_HISTORY_EVENTS = """
 CREATE TABLE IF NOT EXISTS history_events (
-    identifier TEXT NOT NULL PRIMARY KEY,
```

Comment:
you don't need to do that. you can leave this as is and just change type to INTEGER and just not specify it at insertion. Please do that instead as rowid is not really a good column name and also `SELECT * ` does not return it.

Reply:
As discussed in discord we are keeping rowid.